Dockerfile

```diff
@@ -19,5 +21,10 @@ COPY ./ /app/
 
 RUN pip install --no-cache-dir -r /app/requirements.txt
 
+RUN chown -R candig:candig /app
+
+USER candig
+
 RUN touch initial_setup
```

Comment:
```suggestion
RUN touch /app/initial_setup
```

Reply:
Since there's no WORKDIR specified, I think my local build got confused, but adding an absolute path addresses this as well.

Reply:
I  pushed a couple of changes that I think work better: my local machine was having issues without having WORKDIR specified.